src/Rector/Class_/CommandDescriptionToPropertyRector.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Symfony\Rector\Class_;
+
+use PhpParser\Node;
+use PhpParser\Node\Expr\ClassConstFetch;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Scalar\String_;
+use PhpParser\Node\Stmt\Class_;
+use PhpParser\Node\Stmt\Property;
+use PHPStan\Type\ObjectType;
+use PHPStan\Type\StringType;
+use Rector\Core\Rector\AbstractRector;
+use Symplify\RuleDocGenerator\ValueObject\CodeSample\CodeSample;
+use Symplify\RuleDocGenerator\ValueObject\RuleDefinition;
+
+/**
+ * @see https://symfony.com/blog/new-in-symfony-5-3-lazy-command-description
+ *
+ * @see \Rector\Symfony\Tests\Rector\Class_\CommandDescriptionToPropertyRector\CommandDescriptionToPropertyRectorTest
+ */
+final class CommandDescriptionToPropertyRector extends AbstractRector
+{
+    public function getRuleDefinition(): RuleDefinition
+    {
+        return new RuleDefinition('Symfony Command description setters are moved to properties', [
+            new CodeSample(
+                <<<'CODE_SAMPLE'
+use Symfony\Component\Console\Command\Command
+
+final class SunshineCommand extends Command
+{
+    protected static $defaultName = 'sunshine';
+    public function configure()
+    {
+        $this->setDescription('sunshine description');
+    }
+}
+CODE_SAMPLE
+                ,
+                <<<'CODE_SAMPLE'
+use Symfony\Component\Console\Command\Command
+
+final class SunshineCommand extends Command
+{
+    protected static $defaultName = 'sunshine';
+    protected static $defaultDescription = 'sunshine description';
```

Comment:
Is there a way to force this order? Right now its adding the defaultDescription as first property.
So first `$defaultName` and then `$defaultDescription`

Reply:
You'd have to look for existing properties on the `Class_` and if there's a property of "defaultName", add a new property after it.